detection_rules/rule.py

```diff
@@ -583,6 +583,7 @@ class QueryRuleData(BaseRuleData):
     index: Optional[List[str]]
     query: str
     language: definitions.FilterLanguages
+    filters: Optional[List[dict]]
```

Comment:
Was this feature only added in 8.12? Just trying to understand the minstack.

Reply:
This is to not affect older rule versions where the syntax works fine

Reply:
I believe what Mika is discussing here is when was filters first available to use with rules. This way we can min-stack the compatibility within the code for backporting purposes.

Reply:
Oh, assuming that threat intel rules use it, and they were created 3 years ago, I suppose it was added in the early 7.x

https://github.com/elastic/detection-rules/pull/1133